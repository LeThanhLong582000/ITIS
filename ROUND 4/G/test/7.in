7
274
212 18 25 51 135 23 63 157 161 198 176 207 250 55 104 47 213 274 66 236 249 118 10 216 186 6 123 22 115 178 66 250 79 205 37 60 220 212 268 237 17 169 225 231 35 200 92 87 157 240 221 141 122 230 170 83 179 72 249 50 121 158 1 31 219 208 103 81 251 110 231 182 88 84 55 19 154 160 170 193 57 69 29 42 61 195 249 256 20 73 159 38 15 83 184 268 41 20 268 140 83 94 67 84 246 125 186 235 40 103 22 119 105 130 252 80 251 180 66 96 257 264 55 91 26 94 98 5 266 35 267 240 61 143 12 40 155 18 98 158 24 72 110 71 56 118 33 217 152 251 159 154 156 103 175 195 211 68 92 144 47 163 11 99 83 148 103 184 271 79 115 104 54 56 233 270 36 24 141 15 51 112 21 35 259 90 148 185 108 219 89 131 24 4 155 46 206 22 15 245 19 126 184 53 7 36 129 66 253 254 133 162 11 131 227 184 139 45 237 235 246 99 270 58 17 94 164 184 7 47 94 151 218 45 67 128 87 86 109 19 79 40 161 257 172 146 29 162 213 95 274 182 163 204 80 189 274 184 187 176 32 52 119 194 258 117 257 244 1 163 78 175 154 252 
453
298 413 137 160 185 402 221 59 92 285 128 298 164 390 22 335 429 310 323 25 352 352 282 219 423 282 120 150 267 301 392 124 146 426 231 202 194 264 333 183 119 428 258 194 34 57 409 317 122 193 206 334 302 402 322 104 301 426 399 138 425 365 224 245 347 445 313 178 394 402 7 293 422 347 192 282 134 139 153 196 33 226 441 310 387 437 270 153 136 288 30 383 65 264 156 65 123 353 333 123 182 333 234 230 268 56 195 122 430 158 298 128 97 369 181 171 376 146 120 19 33 269 27 251 360 53 432 64 111 101 323 8 265 357 27 114 88 169 369 197 264 13 309 331 158 335 178 349 374 6 161 227 168 166 441 414 138 388 166 1 413 420 214 179 183 363 15 360 382 267 238 73 118 123 152 6 148 43 135 190 378 375 34 423 2 320 395 205 403 92 134 321 266 272 223 314 265 370 273 158 416 91 433 392 98 46 175 262 346 23 118 155 138 414 294 141 137 152 201 143 42 401 30 130 400 238 52 427 66 97 3 79 354 28 277 283 199 21 376 392 304 167 70 322 271 290 233 343 295 349 108 332 337 204 216 122 59 289 364 271 270 58 4 399 195 8 279 387 436 330 369 1 437 425 107 215 92 57 359 167 184 368 26 151 229 250 23 352 350 442 365 364 338 26 392 238 354 97 40 280 452 230 74 374 388 76 324 91 47 109 415 70 104 14 243 84 408 396 129 204 425 74 407 13 380 70 276 96 446 155 41 84 274 294 205 23 165 112 328 313 81 376 379 306 85 191 90 281 111 413 274 109 445 209 305 40 165 232 438 364 62 189 438 129 292 160 232 36 104 25 429 202 399 282 77 196 230 240 348 52 86 435 334 323 342 103 134 5 56 117 215 33 415 327 74 81 114 315 87 41 182 416 231 135 305 80 386 447 311 248 116 275 315 262 130 415 252 80 249 222 258 379 188 404 12 20 23 334 54 57 252 194 160 427 222 241 201 38 323 174 302 388 244 263 339 105 314 356 434 36 290 258 398 
310
68 285 63 250 299 202 206 168 271 259 110 190 52 260 66 97 107 13 226 96 169 139 5 42 170 151 212 198 43 309 263 131 120 277 23 218 46 43 233 263 5 43 12 112 199 138 81 192 182 2 182 190 300 300 294 272 202 211 105 165 45 55 145 253 271 228 6 106 106 83 8 110 120 76 145 173 85 171 155 236 115 109 95 299 302 298 66 27 71 230 80 175 273 31 135 195 187 115 74 276 180 97 104 248 281 16 267 108 185 100 200 208 2 134 179 77 270 61 295 19 211 133 1 57 227 146 179 125 258 290 239 309 270 103 12 168 170 82 144 98 209 285 284 294 251 272 154 101 214 297 224 276 246 212 28 3 180 233 109 213 56 300 231 236 79 125 234 22 169 268 18 208 204 300 71 100 28 302 54 79 36 47 91 248 22 187 292 116 34 65 185 64 63 254 47 256 196 53 211 118 260 292 46 277 56 143 210 283 234 27 302 253 282 257 218 272 42 199 71 81 28 146 72 17 50 279 306 266 278 36 116 191 303 129 211 128 39 249 68 128 237 239 113 61 193 115 64 170 48 242 254 110 299 30 145 108 135 26 36 196 259 102 48 45 215 262 291 67 116 122 227 233 237 3 15 35 284 155 228 13 181 268 111 129 198 233 49 165 121 209 11 305 299 193 276 25 7 64 26 242 82 230 9 36 127 31 151 102 75 230 
102
74 60 92 46 17 17 49 39 45 21 12 46 77 92 15 59 20 98 26 64 35 88 81 50 78 77 82 17 29 9 62 54 43 83 7 68 25 67 64 92 1 47 11 3 81 84 32 80 7 54 18 3 70 75 19 12 55 86 30 46 48 99 23 39 7 42 71 32 13 87 30 101 18 78 86 15 86 8 82 46 64 24 60 9 68 89 50 3 95 48 73 85 76 17 17 58 20 58 11 62 91 26 
349
127 92 49 226 63 176 56 18 234 315 131 118 55 164 13 9 270 103 54 245 13 199 227 120 299 256 156 75 71 283 200 95 146 174 278 2 57 173 76 124 109 243 10 150 8 218 49 237 281 97 318 100 201 324 305 334 31 45 275 110 231 188 106 324 175 106 341 269 293 152 117 244 80 332 83 21 69 41 285 238 60 51 323 258 77 279 236 273 104 5 214 90 292 77 293 277 306 80 167 279 114 220 216 306 169 292 186 13 87 9 208 239 160 244 42 223 340 345 115 214 173 321 199 78 349 297 339 268 193 265 196 92 257 97 298 123 343 272 9 103 301 301 226 112 127 218 179 156 58 223 11 345 320 251 21 77 12 278 99 72 22 250 47 146 302 141 275 140 96 57 46 105 287 79 36 19 342 222 20 287 36 206 95 318 67 85 335 182 179 154 349 108 321 169 133 296 335 345 322 222 43 114 202 346 170 204 337 131 56 227 291 274 45 97 262 200 242 87 153 207 121 47 205 29 293 150 101 137 281 196 58 242 294 135 330 212 155 161 159 272 21 231 304 50 330 36 181 58 62 285 82 264 138 64 192 139 215 233 113 189 111 232 23 19 1 51 153 14 161 103 156 216 334 21 259 1 349 142 34 121 241 298 264 306 60 228 277 325 308 114 25 173 171 300 130 93 5 323 27 274 172 79 177 318 118 222 334 48 80 281 39 268 61 188 244 120 337 94 246 40 215 268 313 332 113 114 250 149 291 204 234 303 250 317 113 261 182 174 176 346 349 156 2 109 70 97 96 234 347 
371
233 211 350 252 99 14 185 317 340 253 61 143 190 362 81 161 208 332 14 37 168 113 149 280 225 221 149 58 6 46 149 315 140 103 25 177 131 107 131 268 68 104 349 322 28 142 252 268 247 67 65 146 182 327 83 21 243 255 194 238 240 342 334 214 206 183 68 337 296 317 129 71 119 226 159 294 120 292 113 23 183 232 266 43 267 202 22 7 304 302 111 269 44 294 154 138 42 206 280 9 344 213 240 29 129 87 191 263 38 7 36 82 93 334 242 334 87 213 39 185 45 163 100 205 59 23 159 72 187 8 273 158 341 116 167 214 139 48 167 323 191 272 65 266 316 197 125 245 117 190 336 109 357 369 270 294 24 203 169 337 200 288 34 141 206 125 197 368 31 269 317 102 29 113 362 71 72 237 91 332 297 258 111 281 244 197 254 255 248 335 63 101 124 214 353 237 48 22 216 277 168 295 121 24 118 58 267 142 250 267 161 297 263 33 188 295 356 88 326 145 126 159 229 93 225 192 279 359 216 42 320 150 124 48 303 138 174 251 147 341 174 69 63 222 26 69 216 230 188 91 95 194 278 247 79 300 225 275 64 3 51 156 325 22 358 70 363 325 291 177 104 164 34 86 246 288 234 31 134 228 205 225 249 273 190 351 27 76 105 192 94 179 66 216 309 47 305 244 174 128 371 295 349 300 104 133 34 216 174 168 343 276 59 313 8 83 210 360 34 364 190 230 92 23 249 177 168 144 50 139 317 217 3 207 310 176 315 180 130 277 17 332 129 115 107 134 44 47 334 8 121 57 252 84 108 356 126 143 216 46 368 175 54 308 118 269 125 313 202 320 107 
382
365 215 40 295 241 7 48 317 164 351 167 197 23 301 117 59 311 206 45 270 271 11 50 133 221 123 135 378 249 284 7 279 337 79 183 109 120 78 257 128 154 308 71 331 338 31 310 226 138 108 121 91 76 211 242 277 255 235 252 114 116 182 241 139 297 268 303 243 23 109 16 144 309 53 52 13 340 32 370 319 333 326 371 202 29 356 339 172 72 62 288 36 296 368 30 173 355 208 216 102 299 79 11 30 184 301 35 178 321 336 302 336 290 38 178 243 271 208 363 245 19 326 6 154 199 358 167 185 16 32 8 103 55 286 360 234 298 185 314 193 370 285 256 169 6 355 28 78 229 277 70 11 196 117 367 207 5 227 17 327 237 129 245 148 239 250 280 176 83 356 261 330 125 74 341 356 81 354 35 144 276 195 62 69 107 258 166 279 164 243 302 324 360 259 164 93 18 149 134 268 297 136 122 125 330 309 2 367 45 202 248 107 266 288 119 153 205 311 103 132 201 312 20 227 237 32 341 253 196 159 174 164 180 157 35 318 56 235 286 104 88 112 291 38 340 55 200 58 284 285 76 310 254 322 381 321 202 87 245 118 341 46 127 139 367 307 348 157 233 221 344 328 23 179 61 99 270 335 339 68 176 174 95 72 197 161 38 287 50 329 183 136 187 207 256 350 163 244 326 201 123 349 236 253 12 355 10 213 205 353 286 359 310 101 240 20 296 176 160 140 118 120 42 46 319 350 97 30 98 58 346 243 377 283 174 166 146 113 140 265 303 202 340 336 323 129 324 379 76 232 153 87 71 368 297 357 352 65 93 312 269 54 72 23 139 292 368 34 203 36 72 232 3 6 260 355 4 283 52 172 140 48 
