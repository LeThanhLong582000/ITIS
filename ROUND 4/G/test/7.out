130
229
154
50
174
183
195
